insights/combiners/httpd_conf.py

```diff
@@ -66,6 +66,7 @@
 
 from insights.core.plugins import combiner
 from insights.parsers.httpd_conf import HttpdConf, dict_deep_merge, ParsedData
+from insights.parsers.httpd_conf import HttpdConfSclHttpd24, HttpdConfSclJbcsHttpd24
 
 
 @combiner(HttpdConf)
```

Comment:
I *think* that the common class from which all three resulting combiners inherit should *not* be decorated with HttpdConf.

Reasons:

1. Each combiner should process exactly one set of config files. The combiner doesn't work correctly with more than 1 httpd.conf (the main file that contains the include most probably).

2. I remember this being a problem in the past. IDK whether it is problematic now.